include/main.h

```diff
@@ -1,64 +1,58 @@
+char const* const SECOND = "________\n |/   |\n |   (_)\n |    |\n |    |\n |\n_|_\n";
+char const* const THIRD = "________\n |/   |\n |   (_)\n |   /|\n |    |\n |\n_|_\n";
+char const* const FOURTH =  "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |\n_|_\n";
+char const* const FIFTH = "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   /\n_|_\n";
+char const* const SIXTH = "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   / \\\n_|_\n";
 
-// hangman drawing based on how many lives lost
-// using macros because initiating an array of pointers is not a constant expression, which can't be initialized as a global variable
-#define FRESH "________\n |/   |\n |\n |\n |\n |\n_|_\n"
-#define FIRST "________\n |/   |\n |   (_)\n |\n |\n |\n_|_\n"
-#define SECOND "________\n |/   |\n |   (_)\n |    |\n |    |\n |\n_|_\n"
-#define THIRD "________\n |/   |\n |   (_)\n |   /|\n |    |\n |\n_|_\n"
-#define FOURTH "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |\n_|_\n"
-#define FIFTH "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   /\n_|_\n"
-#define SIXTH "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   / \\\n_|_\n"
+char const* const stages[7] = {FRESH, FIRST, SECOND, THIRD, FOURTH, FIFTH, SIXTH};
 
 // dynamic variables
-char *word; // holds the random word chosen at the beginning of the game
-char guess[2]; // holds the 1 letter guess
+char const* word; // holds the random word chosen at the beginning of the game
 char command[2]; // command placeholder for when user wins/loses the game
 char difficulty_input[8]; // holds user's input when choosing difficulty
-short int incorrect_count = 0; // tracks the number of incorrect guesses
-short int match_flag; // used to track whether guess was a hit or a miss; 0 = miss, 1 = has at least one hit
+short incorrect_count = 0; // tracks the number of incorrect guesses
+short match_flag; // used to track whether guess was a hit or a miss; 0 = miss, 1 = has at least one hit
 
 int randomDictionary; // used in pickWord()
 int wordLength; // used in prompt()
-char *randomWord; // contains the random word
-short int output_boolean[51]; // the boolean values for each guessing spot (handled as 0 or 1)
+short output_boolean[51]; // the boolean values for each guessing spot (handled as 0 or 1)
 char output_display[51]; // the underscores that will be dynamically updated as the user guesses correctly
 
-const char *stages[7] = {FRESH, FIRST, SECOND, THIRD, FOURTH, FIFTH, SIXTH};
-const char alphabet[26] = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'}; // constant alphabet source
 char letterBank[26]; // dynamically display which letters have been used and which are available
-short int letterIndex; // index of the char based on ASCII; (int)guess - 65, since A = 65
+short letterIndex; // index of the char based on ASCII
 
-typedef enum difficulty {
+typedef enum Difficulty {
```

Comment:
Stylistic convention: 
Structs/enums ("types") should be PascalCase.
Or, another pattern oft-used is snake_case with pre/post-fixes (e.g. `difficulty_t`).